src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+@import './blocks/star';
+@import './blocks/card';
+
+:root {
+  --link-color: #00acdc;
+  --color: #616070;
+  --bg-color: #fff;
+  --text-color: #060b35;
```

Comment:
If you use scss create only scss variables, not CSS variable

Reply:
done